core/domain/suggestion_services.py

```diff
@@ -216,17 +215,17 @@ def get_all_stale_suggestion_ids():
     )
 
 
-def _update_suggestion(suggestion):
+def update_suggestion(suggestion):
     """Updates the given suggestion.
 
     Args:
         suggestion: Suggestion. The suggestion to be updated.
     """
 
-    _update_suggestions([suggestion])
+    update_suggestions([suggestion])
 
 
-def _update_suggestions(suggestions, update_last_updated_time=True):
+def update_suggestions(suggestions, update_last_updated_time=True):
```

Comment:
Does this one need to also be made public, or just the ``update_suggestion`` above?